src/App.js

```diff
@@ -1,15 +1,17 @@
-import React, { Component } from 'react';
-import DataTable from './DataTable';
-import './App.css';
+import React from "react";
 
```

Comment:
Both machines and human love functional component, more readable, less buggy they are.